src/duckstation-qt/mainwindow.cpp

```diff
@@ -297,6 +297,7 @@ std::optional<WindowInfo> MainWindow::acquireRenderWindow(RenderAPI render_api,
     }
 
     updateDisplayWidgetCursor();
+    updateDisplayRelatedActions(!surfaceless, fullscreen);
```

Comment:
this might no longer be needed after b07998512e4222317d54de9343c58f166af80a1f

Reply:
Yep, it should get called through destroy and create.

Reply:
Actually this code path doesn't recreate the display widget, so we still need to call `updateDisplayRelatedActions`. Fixed in #3485.